curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/use-bracket-notation-to-find-the-nth-character-in-a-string.english.md

```diff
@@ -38,15 +38,11 @@ tests:
 <div id='js-seed'>
 
 ```js
-// Example
-var firstName = "Ada";
-var secondLetterOfFirstName = firstName[1];
```

Comment:
Same problem with the hint.

Reply:
Good catch. I will try to move the example into the description section.

Reply:
I moved the example to the challenge description.